pkg/yang/entry_test.go

```diff
@@ -3173,6 +3338,29 @@ func TestDeviation(t *testing.T) {
 				},
 			}},
 		},
+	}, {
+		// TODO(wenovus): Support deviate delete for leaf-lists once its semantics are clear.
+		// https://github.com/mbj4668/pyang/issues/756
+		desc: "error case - deviation delete on a leaf-list",
+		inFiles: map[string]string{
+			"deviate": `
+				module deviate {
+					prefix "d";
+					namespace "urn:d";
+
+					leaf-list a {
+						type string;
+						default "fish";
+					}
+
+					deviation /a {
+						deviate delete {
+							default "fishsticks";
+						}
+					}
+				}`,
+		},
+		wantProcessErrSubstring: "deviate delete on default statements unsupported for leaf-lists",
```

Comment:
Potentially we can remove this TODO and address the issue here if you agree with my assessment of 7950 above.

Reply:
Can you see my reply above to see if you agree with what I intend to do for this here?

Reply:
Yep, that sounds good to me.